psi4/src/psi4/occ/mograd.cc

```diff
@@ -129,12 +129,12 @@ void OCCWave::mograd() {
         rms_wogA = 0;
         for (int i = 0; i < nidpA; i++) rms_wogA += wogA->get(i) * wogA->get(i);
         norm_wogA = std::sqrt(rms_wogA);
-        rms_wogA = std::sqrt(rms_wogA) / nidpA;
+        rms_wogA = std::sqrt(rms_wogA / nidpA);
 
         rms_wogB = 0;
         for (int i = 0; i < nidpB; i++) rms_wogB += wogB->get(i) * wogB->get(i);
         norm_wogB = std::sqrt(rms_wogB);
-        rms_wogB = std::sqrt(rms_wogB) / nidpB;
+        rms_wogB = std::sqrt(rms_wogB / nidpB);
```

Comment:
this may win "most consternation caused by misplaced parenthesis"

Reply:
Oh wow.